jenkins/scripts/integration_test.sh

```diff
@@ -36,12 +36,7 @@ UPDATED_REPO="${UPDATED_REPO:-https://github.com/${REPO_ORG}/${REPO_NAME}.git}"
 UPDATED_BRANCH="${UPDATED_BRANCH:-master}"
 CAPI_VERSION="${CAPI_VERSION:-v1alpha3}"
 
-if [ "${IMAGE_OS}" == "Ubuntu" ]
-then
-  DEFAULT_HOSTS_MEMORY="${DEFAULT_HOSTS_MEMORY:-6144}"
-else
-  DEFAULT_HOSTS_MEMORY="${DEFAULT_HOSTS_MEMORY:-8192}"
-fi
+DEFAULT_HOSTS_MEMORY="${DEFAULT_HOSTS_MEMORY:-8192}"
```

Comment:
I think you need to change also `TARGET_NODE_MEMORY` in capi_bm_v1aX_integration_tests_ubuntu.yaml in gerrit, which is defaulted to 4096.

Reply:
Because the initial value is inherent from from here `TARGET_NODE_MEMORY` in [here](https://github.com/metal3-io/project-infra/blob/ad99716fcf645498ff133880f87b9978dbaa89c5/jenkins/jobs/integration_tests.pipeline#L47).

Reply:
hum, maybe we just override it for now, as it is hopefully a temporary workaround until we have raw image streaming in place. Otherwise, the TARGET_NODE_MEMORY parameter does not make sense anyways, since we just base the size on the target distribution

Reply:
very good catch, thank you